manifests/application.pp

```diff
@@ -34,6 +35,7 @@
   $post_sync_url = "${::foreman::foreman_url}${deployment_url}/api/v2/repositories/sync_complete?token=${post_sync_token}"
   $candlepin_ca_cert = $::certs::ca_cert
   $pulp_ca_cert = $::certs::katello_server_ca_cert
+  $crane_ca_cert = $::certs::katello_server_ca_cert
```

Comment:
This can probably also be a `Stdlib::Absolutepath`. I also think this should be kept together with the URL setting since they're related. We should have done the same with the other settings but that can be a separate PR.

Reply:
@ekohl I don't understand what is meant to change here.

Reply:
Ignore this one, I was mistakenly thinking that it was a class parameter.